assets/_custom.scss

```diff
@@ -31,6 +31,10 @@ table.data-table {
     &.icon-information {
```

Comment:
You need to allow the research docs to allow raw HTML. In `infonline/i00.md`, we're using a `<details>` tag but that is broken in the rendered output:

![image](https://github.com/tweaselORG/tracker-wiki/assets/4048809/d90d0333-af81-4f98-b40c-e9c35ae42703)